src/lib.rs

```diff
@@ -0,0 +1,58 @@
+mod actor;
+mod lines_to_params;
+mod parser;
+mod utils;
+
+use pyo3::prelude::*;
+
+/// Formats the sum of two numbers as string.
+#[pyfunction]
+fn sum_as_string(a: usize, b: usize) -> PyResult<String> {
+    Ok((a + b).to_string())
+}
+
+/// A Python module implemented in Rust.
+#[pymodule]
+fn nari_act_rust(py: Python, m: &PyModule) -> PyResult<()> {
+    m.add_function(wrap_pyfunction!(actor::parse_actor, m)?)?;
+    m.add_function(wrap_pyfunction!(lines_to_params::ability_from_params, m)?)?;
+    m.add_function(wrap_pyfunction!(
+        lines_to_params::action_effect_from_params,
+        m
+    )?)?;
+    m.add_function(wrap_pyfunction!(
+        lines_to_params::status_effect_from_params,
+        m
+    )?)?;
+    m.add_function(wrap_pyfunction!(
+        lines_to_params::statuslist_from_params,
+        m
+    )?)?;
+
+    let parser_module = PyModule::new(py, "parser")?;
+    parser_module.add_function(wrap_pyfunction!(parser::param_to_2_byte_int, m)?)?;
+    parser_module.add_function(wrap_pyfunction!(parser::param_to_2x2_byte_int, m)?)?;
+    parser_module.add_function(wrap_pyfunction!(parser::param_to_4_byte_float, m)?)?;
+    parser_module.add_function(wrap_pyfunction!(parser::param_to_4_byte_int, m)?)?;
+    parser_module.add_function(wrap_pyfunction!(parser::param_to_4x1_byte_int, m)?)?;
+    parser_module.add_function(wrap_pyfunction!(parser::params_to_8_byte_int, m)?)?;
```

Comment:
Any chance we could name these closer to "standard"? Like `param_to_4_byte_float` is really what, `param_to_float32`? The array sequence ones might be weirder hm.

Reply:
I like the idea of `f32_from_params` and maybe we could do something along the lines of `u8_4_tuple_from_params` or `u8x4_from_params` instead of `param_to_4x1_byte_int`
What ever feels more correct in the feel of the naming.

Reply:
I think we'd best stick to python names for now, or otherwise rust in rust and python in python but have pyo3 remap the function names. It's clearer to read when the name matches the signature too, so I'm thinking `params_to_x` type names.

We also don't need the `u64` stuff, ACT doesn't use `u64` for anything besides timestamps technically, so at most we'd need `u32`. If anything, we could *only* do `u32` and have the smaller functions dynamically scope.